draft-ietf-rtcweb-jsep.xml

```diff
@@ -1880,7 +1889,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               the "active" role is RECOMMENDED.</t>
 
               <t>If present in the offer, an "a=rtcp-mux" line, as specified
-              in <xref target="RFC5761"></xref>, Section 5.1.1.</t>
+              in <xref target="RFC5761"></xref>, Section 5.1.1. If the
+              "max-mux" RTCP multiplexing policy is set and no "a=rtcp-mux"
+              line is present in the offer,  then the m=line MUST be
+              marked as rejected by setting the port in the m= line to zero, as
+              indicated in <xref target="RFC3264"></xref>, Section 6.
+              </t>
```

Comment:
The RTCP case breaks in the case where the answerer wants to force RTCP mux, receives an offer from a non-muxer (which results in a non-mux m= section), and then later reoffers with a new m= section. Unless the answerer is somehow keeping track that the remote side might not want to do mux, it will send its reoffer with the new m= section being mux-only.

I now think that the BUNDLE case is distinct, there never is any ambiguity on either side. If the answerer doesn't support BUNDLE, you just won't use BUNDLE, and that's easy to deal with.
Anyway, TL;DR - I think this text LG as-is. It is a corner case.